guides/theming.md

```diff
@@ -251,3 +251,27 @@ function changeTheme(themeName) {
 ### Theming your own components
 For more details about theming your own components,
 see [theming-your-components.md](./theming-your-components.md).
+
```

Comment:
Should we mention the `mat-focus-indicator` class somewhere?

Reply:
Added a sentence & example towards the end, let me know if you had something else in mind.